bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/configuration/ICloudAccountThingConfiguration.java

```diff
@@ -12,12 +12,15 @@
  */
 package org.openhab.binding.icloud.internal.configuration;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
 /**
  * Represents the configuration of an iCloud Account Thing.
  *
  * @author Patrik Gfeller - Initial Contribution
  *
  */
+@NonNullByDefault
 public class ICloudAccountThingConfiguration {
     public String appleId;
```

Comment:
If there is a default, add it here (BTW: is it really needed that this is `Integer` instead of `int`?)

```
public Integer refreshTimeInMinutes = 5;
```

else annotate the with `@Nullable`:

```
public @Nullable String appleId;
```

Reply:
I think it can be int. I will try to change it and see if the build will succeed.